docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -1,168 +1,367 @@
 
-## Step 4/4 Backport your changes
+    ```code
+    $ git switch -c my-branch
+    ```
 
-Since our docs site is versioned, we want to ensure that any changes you make to
-the most recent version of our documentation are reflected for the versions of
+    You now have a branch for a specific version of the documentation content.
+    Before making any changes, note how the content is organized so you know where to find the files 
+    to update.
+    
+    Within the `content/*.*` directory in your branch, there's a `docs` subdirectory. 
+    The `content/*.*/docs` subdirectory contains the files and folders used to produce a specific version 
+    of the documentation. 
+    
+    The subdirectory includes the following:
+
+    - The `img/` folder for images used inside the pages.
+    - The `pages/` folder for the Markdown files that contain documentation content. 
+      Every file in this folder is rendered as a page.
+    - The `pages/includes` folder for content that's only rendered when it's included in the body of a page.
+    - The `config.json` file for configuring version information.
+
+    You're now ready to make changes in your own branch using the code editor of your choice.
+
+## Step 3/5. Test changes locally
+
+As you make changes to the content in your local branch, it's helpful to see how the changes will be 
+rendered when the documentation is published.
+
+You can use `yarn` to run a development server that detects changes to the documentation and displays 
+them whenever you manually refresh the page in the browser.
+You can also use `yarn` commands to check for errors and fix common issues.
+
+To test changes in your local environment:
+
+1. Change to the root of the `docs` directory by running the following command:
+    
+    ```code
+    $ cd docs
+    ```
+
+1. Start the development server by running the following commands:
+    
+    ```code
+    $ yarn dev
+    ```
+
+1. Open a web browser and navigate to the documentation using the URL `localhost:3000/docs`. 
+
+1. Validate the Markdown syntax by running the following commends:
+
+    ```code
+    # Check your Markdown syntax
+    $ yarn markdown-lint
+    # markdown-lint, but also checks that external links work
+    $ yarn markdown-lint-external-links
+    ```
+
+    For more information about using `yarn` commands to work with Teleport documentation, 
+    see the [README](https://github.com/gravitational/docs).
+
+## Step 4/5. Create a pull request
+
+If you're an internal contributor with permission to access the [Teleport repository](https://github.com/gravitational/teleport), you can commit changes and push branches directly to the repository.
+If you're an external contributor, you can commit changes to your private branch and push private branches to a fork of the [Teleport repository](https://github.com/gravitational/teleport).
+
+### Internal contributors
```

Comment:
Should we use `Tabs` for the "Internal contributors" and "External contributors" instructions?

Reply:
I think the tabs and boxes are way overused throughout the docs personally.